compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3079,6 +3079,67 @@ object Parsers {
 
  /* -------- PARAMETERS ------------------------------------------- */
 
+    /** DefParamClause ::= DefTypeParamClause 
+     *                   | DefTermParamClause 
+     *                   | UsingParamClause
+     */
+    def typeOrTermParamClause(
```

Comment:
This method appears to be unused

Reply:
It indeed is, I left it for completeness, should I remove it ?